src/compressed_tensors/quantization/lifecycle/compressed.py

```diff
@@ -42,7 +42,8 @@ def compress_quantized_weights(module: Module):
         # no quantization scheme or weights not quantized, nothing to do
         return
 
-    if scheme is QuantizationStatus.COMPRESSED:
+    status = getattr(module, "quantization_status", None)
+    if status is QuantizationStatus.COMPRESSED:
```

Comment:
See above, theoretically this should be moved to within `apply_quantization_status`

Reply:
Should this function be removed? Along with 

[examples/quantize_and_pack_int4.ipynb](https://github.com/neuralmagic/compressed-tensors/blob/c6491592f9655a4197c2fc17335f6c0936b1ab74/examples/quantize_and_pack_int4.ipynb#L29)?